src/lib/fileUploader/gcpBucket.ts

```diff
@@ -68,7 +67,7 @@ export const saveFiles = (bucket: Bucket, files: File[], options: Options) => {
                         .pipe(remoteFile)
                         .on('error', reject)
                         .on('finish', resolve)
-                ).then(() => fileId);
+                ).then(() => ({ fileId, originalFile: file.originalFile }));
```

Comment:
I think that using wait here would increase readability [#6](https://github.com/AckeeCZ/unicore/pull/6#discussion_r334331643)

Reply:
Please just make sure that all the types work correctly. If I am not mistaken, the function now returns a new type (file). Test it works even when the types are not installed in the project using unicore (e.g. they are not just in dev dependencies).  [#6](https://github.com/AckeeCZ/unicore/pull/6#discussion_r334331643)